payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -1435,6 +1437,27 @@ class StripeApiRepository @JvmOverloads internal constructor(
         )
     }
 
+    override suspend fun listPaymentDetails(
+        clientSecret: String,
+        paymentMethodTypes: Set<String>,
```

Comment:
is this parameter needed? It looks like it's only set to emptySet and is otherwise unused (I could be missing something though)

Reply:
It is. I was supposed to add it to the other interfaces as well. Just pushed a fix.